<ui version="4.0" >
 <class>FormP</class>
 <widget class="QWidget" name="FormP" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>431</width>
    <height>429</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>431</width>
    <height>429</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Właściwości</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QTextEdit" name="textEdit" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>256</y>
     <width>413</width>
     <height>128</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="html" >
    <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Rodzaje typów:&lt;/span>&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">"&lt;span style=" font-weight:600;">file&lt;/span>" - podana jest ścieżka do zwykłego pliku&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">"&lt;span style=" font-weight:600;">plugin:param.x&lt;/span>" - odtwarzane specjalnie przez plugin ( param to parametr przesłany do wtyczki, a x to nazwa wtyczki ). Może służyć do generowania jakiś dźwięków przez plugin lub odczyt np. audio CD.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">"&lt;span style=" font-weight:600;">internet:x&lt;/span>" - strumień otwierany z internetu ( x to nazwa wtyczki ). Obsługiwany protokół i format strumienia zależy od wtyczki.&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QLabel" name="titleL" >
   <property name="geometry" >
    <rect>
     <x>64</x>
     <y>9</y>
     <width>358</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>358</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>358</width>
     <height>18</height>
    </size>
   </property>
   <property name="text" >
    <string>Title flom QMP playlist</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>57</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Typ pliku:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>226</y>
     <width>104</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Wpisz adres:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="adresE" >
   <property name="geometry" >
    <rect>
     <x>119</x>
     <y>226</y>
     <width>303</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OKB" >
   <property name="geometry" >
    <rect>
     <x>347</x>
     <y>393</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>49</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Tytuł:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>89</y>
     <width>413</width>
     <height>126</height>
    </rect>
   </property>
   <property name="title" >
    <string>Wybierz typ pliku</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="1" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Wtyczka:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QRadioButton" name="fileB" >
      <property name="text" >
       <string>Plik</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QRadioButton" name="adresB" >
      <property name="text" >
       <string>Adres internetowy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QComboBox" name="pluginsB" />
    </item>
    <item row="1" column="0" >
     <widget class="QRadioButton" name="pluginB" >
      <property name="text" >
       <string>Wtyczka</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QPushButton" name="applyTB" >
      <property name="text" >
       <string>Zastosuj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="applyB" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>393</y>
     <width>328</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Zastosuj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="typeE" >
   <property name="geometry" >
    <rect>
     <x>101</x>
     <y>57</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>32</y>
     <width>49</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Numer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nrL" >
   <property name="geometry" >
    <rect>
     <x>64</x>
     <y>32</y>
     <width>49</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>32</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Rozmiar pliku:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fSizeL" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>32</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>0MB</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>typeE</tabstop>
  <tabstop>fileB</tabstop>
  <tabstop>pluginB</tabstop>
  <tabstop>adresB</tabstop>
  <tabstop>pluginsB</tabstop>
  <tabstop>applyTB</tabstop>
  <tabstop>adresE</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>applyB</tabstop>
  <tabstop>OKB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
